addons/portal/models/res_partner.py

```diff
@@ -18,3 +17,10 @@ def can_edit_vat(self):
         edit it (as in backend)."""
         self.ensure_one()
         return not self.parent_id
+
+    @api.model
+    def _get_portal_partner_from_context(self):
```

Comment:
As posted on other comments, I think we should be able to get rid of this method for now.